### PR TITLE
fix: globalsearch styles and documentation widget links

### PR DESCRIPTION
**Pull request by ShmuelLotman**
Closes #

This addresses the changes needed to docs panel styling and also moves the useful links to make room for the upcoming pinned items

### CODE REVIEW

File: src/me/components/DocSearchWidget.scss [hunk 1]
```diff
@@ -44,6 +44,7 @@ $article-text: #d4d7dd;
   }
   .DocSearch-Container {
     background: transparent;
+    max-height: calc(100vh -500px) !important;
```
Comment (Asalem1) [inline src/me/components/DocSearchWidget.scss:47]: should we set a min-height too?

File: src/me/containers/MePage.tsx [hunk 2]
```diff
@@ -118,7 +117,6 @@ export class MePage extends PureComponent<Props> {
                       <GettingStarted />
                     </Panel.Body>
                   </Panel>
-                  <Docs />
```
Comment (Asalem1) [inline src/me/containers/MePage.tsx:121]: where's this at now?

Comment (ShmuelLotman) [reply src/me/containers/MePage.tsx:121]: adding this back in

File: src/me/containers/MePage.scss [hunk 3]
```diff
@@ -31,9 +31,9 @@
 
 .tutorials-list > li {
   font-size: 1.25em;
-  background-color: $g2-kevlar;
+  background-color: transparent;
   border-radius: $radius;
-  padding: $ix-marg-c;
+  padding: 8px;
```
Comment (Asalem1) [inline src/me/containers/MePage.scss:36]: 😢

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 58 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2087)

File: src/me/components/DocSearchWidget.scss (hunk 1 above)
Comment (ShmuelLotman) [reply src/me/components/DocSearchWidget.scss:47]: not needed

File: src/me/containers/MePage.tsx [hunk 4]
```diff
@@ -16,11 +16,10 @@ import {
   ComponentColor,
 } from '@influxdata/clockface'
 import Resources from 'src/me/components/Resources'
-import Docs from 'src/me/components/Docs'
 import GettingStarted from 'src/me/components/GettingStarted'
 import RateLimitAlert from 'src/cloud/components/RateLimitAlert'
 import AlertsActivity from 'src/me/components/AlertsActivity'
-
+import Docs from 'src/me/components/Docs'
```
Comment (genehynson) [inline src/me/containers/MePage.tsx:22]: nit: undo this change

**Review by genehynson — APPROVED** [review]

**Review by Asalem1 — APPROVED** [review]
